eng/common/pipelines/templates/steps/cosmos-emulator.yml

```diff
@@ -1,7 +1,7 @@
 parameters:
   EmulatorInstallPath: "$(Agent.HomeDirectory)/../../Program Files/Azure Cosmos DB Emulator/Microsoft.Azure.Cosmos.Emulator.exe"
   EmulatorMsiUrl: "https://aka.ms/cosmosdb-emulator"
-  StartParameters: ''
+  StartParameters: '/noexplorer /noui /enablepreview /EnableSqlComputeEndpoint /SqlComputePort=9999 /disableratelimiting /partitioncount=50 /consistency=Strong'
```

Comment:
I think you need to revert the changes to this right here. This is coming from `eng/common` and you need to follow a different workflow to merge changes to files under that path. (PR to sdk-tools, etc)

However, the change of this default is un-necessary, you providing the `EmulatorStartParameters` will definitely be good enough to make this work.